packages/cli/test/functional/updatetest.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
+const fs = require('fs-extra-promise');
+const { execSync } = require('child_process');
+
+const { cleanupConvert } = require('./testUtil/cleanup');
+
+const {
+  testSites,
+  testConvertSites,
+  testTemplateSites,
+} = require('./testSites');
+
+/* eslint-disable no-console */
+
+function printFailedMessage(err, siteName) {
+  console.log(err);
+  console.log(`Failed to update: ${siteName}`);
+}
+
+const execOptions = {
+  stdio: ['inherit', 'inherit', 'inherit'],
+};
+
+testSites.forEach((siteName) => {
+  console.log(`Updating ${siteName}`);
+  try {
+    execSync(`node ../../index.js build ${siteName} ${siteName}/expected`, execOptions);
+  } catch (err) {
+    printFailedMessage(err, siteName);
+    process.exit(1);
+  }
+});
+
+testConvertSites.forEach((siteName) => {
+  console.log(`Updating ${siteName}`);
```

Comment:
Similarly, I think we can abstract this into a separate function.

Reply:
There's also quite a bit of overlap with the functions used to generate the output in `test.js`. Perhaps we should generalize the functions that performs the build. We can pass in the optional argument for where the built files should be output. I think that would allow us to use it in both `test` and `updatetest`.